perl.c

```diff
@@ -2859,7 +2860,7 @@ Perl_get_av(pTHX_ const char *name, I32 flags)
 
     PERL_ARGS_ASSERT_GET_AV;
 
-    if (flags)
+    if (flags & ~SVf_UTF8)
```

Comment:
Maybe add a comment that this is logically the same as  `(flags && flags != SVf_UTF8)`

Reply:
I could do but I think I'd find that style more confusing than currently :)

Reply:
I don't understand how adding a *comment* that the two are equivalent could be more confusing than not having the comment. A lot of people dont think bit flips properly. But whatever, ill do a follow up PR.